libsoftwarecontainer/component-test/networkgateway_componenttest.cpp

```diff
@@ -46,6 +46,8 @@ class NetworkGatewayTest : public SoftwareContainerGatewayTest
         srand(time(NULL));
         uint32_t containerID = rand() % 100;
 
+        ::testing::DefaultValue<bool>::Set(true);
```

Comment:
There seems to be a slightly different behavior now that ReturnCode isn't used. I had a failing test case which stopped failing once I added this line, which sets the default return value for mocked functions.